src/opnsense/mvc/app/views/OPNsense/Diagnostics/log.volt

```diff
@@ -44,14 +47,19 @@
               },
               requestHandler: function(request){
                   if ( $('#severity_filter').val().length > 0) {
-                      request['severity'] = $('#severity_filter').val();
+                      let severities = ["Emergency", "Alert", "Critical", "Error", "Warning", "Notice", "Informational", "Debug"];
+                      let selectedSeverity = $('#severity_filter').val();
+                      request['severity'] = severities.slice(0,severities.indexOf(selectedSeverity) + 1);
```

Comment:
@kulikov-a not sure if I like this part, is there a specific reason to duplicate the levels here?

Reply:
@AdSchellevis Hi! me neither) i can map select options values to array? (one time at page load).
`var severities = $('#severity_filter option').map(function(){  return this.value; });`
i need array here to make `request['severity']` array from one selected "max severity" value